daemon/algod/api/server/v2/test/handlers_test.go

```diff
@@ -333,7 +333,7 @@ func getPendingTransactionsTest(t *testing.T, format string, max uint64, expecte
 func TestPendingTransactionLogsEncoding(t *testing.T) {
 	partitiontest.PartitionTest(t)
 
-	response := generated.PendingTransactionResponse{
+	response := generatedV2.PendingTransactionResponse{
```

Comment:
this was making the linter fail because there were two imports. the same package (one with an alias the other without) and it was non-deterministic without this update.